moped-etl/arcgis/utils.py

```diff
@@ -11,6 +11,7 @@
 AGOL_USERNAME = os.getenv("AGOL_USERNAME")
 AGOL_PASSWORD = os.getenv("AGOL_PASSWORD")
 HASURA_ENDPOINT = os.getenv("HASURA_ENDPOINT")
+HASURA_SQL_ENDPOINT = os.getenv("HASURA_SQL_ENDPOINT")
```

Comment:
This will require a PR to airflow to include this new environment variable. I guess it could be inferred from the `HASURA_ENDPOINT`, but this felt cleaner to define like everything else.